scenarios3/10_Gehenna.cfg

```diff
@@ -233,8 +233,12 @@
     {RECALL_HEROES}
     [event]
         name=turn 2
+#ifdef EASY
         {RARE_ITEM 21 3}
+#endif
+#ifndef HARD
         {RARE_ITEM 22 3}
+#endif
         {RARE_ITEM 23 3}
         {RARE_ITEM 21 2}
```

Comment:
Can you please replace them by less valuable items instead of removing them entirely? It sucks to discover a hidden room and find nothing in there.

Reply:
I can, but it's not that there's nothing in there.  There's still 4 items on easy, 3 on normal, and 2 on hard.  And a chest.

Reply:
Aha, yeah, sorry, I didn't notice they were all close to each other.